sdk/core/Azure.Core.TestFramework/src/RecordedTestBase.cs

```diff
@@ -298,8 +298,7 @@ protected object InstrumentMgmtOperation(Type operationType, object operation, M
                 new[] { typeof(IInstrumented) },
                 operation,
                 managementInterceptor,
-                new GetOriginalInterceptor(operation),
```

Comment:
What if someone was calling this directly? Would they get operation instrumentation now?